doc/src/modules/polys/basics.rst

```diff
@@ -14,6 +14,319 @@ polynomials within SymPy. All code examples assume::
     >>> x, y, z = symbols('x,y,z')
     >>> init_printing(use_unicode=False, wrap_line=False, no_global=True)
 
+Basic concepts
+==============
+
+Polynomials
+-----------
+
+Given a family `(x_i)` of symbols, or other suitable objects,
+expressions derived from them by repeated addition, subtraction
+and multiplication are called *polynomial expressions in the
+generators* `x_i`.
```

Comment:
I see, so they are included in the x_i. In that case, maybe mention "or other suitable objects, including numbers" or something like that. I think I understand what you are defining here, but it's easy to confuse this with what people usually think of as a "polynomial".